modules/ROOT/pages/apikit-odatav4-extension-reference.adoc

```diff
@@ -218,50 +206,51 @@ Default configuration
 
 === For Configurations
 
-* <<Config>> {nbsp}
+* <<Config>> 
 
 === Throws
 
-* APIKIT-ODATA:INTERNAL_SERVER_ERROR {nbsp}
-* APIKIT-ODATA:MALFORMED_SEARCH {nbsp}
-* APIKIT-ODATA:MALFORMED_URI {nbsp}
-* APIKIT-ODATA:NOT_IMPLEMENTED {nbsp}
-* APIKIT-ODATA:TRANSFORMATION {nbsp}
-
-
+* APIKIT-ODATA:INTERNAL_SERVER_ERROR 
+* APIKIT-ODATA:MALFORMED_SEARCH 
+* APIKIT-ODATA:MALFORMED_URI 
+* APIKIT-ODATA:NOT_IMPLEMENTED 
+* APIKIT-ODATA:TRANSFORMATION 
+* APIKIT-ODATA:EXPANSION
```

Comment:
```suggestion
```

Sorry for mistakenly saying that you should add an error here. I commented on the docs for the wrong operation!